modules/config-posture/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "is_organizational" {
+  type        = bool
+  default     = false
+  description = "(Optional) True/False whether secure-for-cloud should be deployed in an organizational setup"
+}
+
+variable "region" {
+  type        = string
+  default     = "us-sanjose-1"
+  description = "(Optional) Default region for resource creation"
+}
```

Comment:
Do we need the `region` as input to the module? since we aren't using it and I think it is mainly consumed in the oci provider block.

Reply:
Actually we don't - I think we can get rid off this. I added initially as part of the boilerplate but this can be removed

thanks!